libs/domain/product/src/services/adapter/normalizers/facet-range/facet-range.normalizer.ts

```diff
@@ -7,8 +7,8 @@ export function facetsRangeNormalizer(
   rangeFacets: ApiProductListModel.RangeFacet[]
 ): RangeFacet[] {
   return rangeFacets.reduce((normalizedFacetList: RangeFacet[], facet) => {
-    //ignore the facet if there is no difference between min and max
-    if (facet.min === facet.max) {
+    //ignore the facet if difference between min and max is 1
+    if (facet.max - facet.min <= 1) {
```

Comment:
Nope, BE does not care about step. The idea was to have at least the integer to prevent floating values. In result steps of the range slider controlled by the browser automatically depends on available screen width. For more accurate interaction user has min and max inputs